### DISCUSSION TITLE
MSのOAuthの追加リクエストの意図を教えてください

### DISCUSSION THREAD

**Discussion by koron** [vim-jp/vim-jp.github.io#398] [Q&A] [answered]
@U-MA さんから、MSのリードオンリーのOAuthアクセス追加のリクエストがありました。
これを追加するべきかの判断ができずに困っています。

いま現在どのようなことに困っていて、
このOAuthアクセスを追加することでどのようにそれが解決されるのか
について @U-MA さんに説明をお願いできないでしょうか?

説明が無い場合は、拒絶しますのであしからず。

**Comment by U-MA** [comment, marked as answer]
@koron さん
今、このメンションに気づきました。反応遅くなり申し訳ございません。

Azure Repos のリポジトリに対し、git fetch を実施したところ、
Git Credential Manager が表示されたので Github と連携したアカウントで認証を行っていたのですが
その際に追加の権限として Github の Organization access に対しても追加の権限を求めているようでした。
それに気づかず、認証ボタンを押下していたため vim-jp 側に MS のリードオンリーのアクセス追加リクエストが飛んでしまったのだと思います。

@koron さん、および vim-jp の皆さんにご迷惑をおかけして申し訳ありません。
権限リクエストについては拒絶していただいて構いません。

**Reply by koron** [reply]
@U-MA さん

確認をありがとうございます。
誤操作によるもので拒否(Deny)して良いものであるため
さっそくDenyしました。

**Comment by koron** [comment]
要求理由が誤操作と分かったため、当該リクエストはDenyしました。

本件はこれで閉じます。
